### PR TITLE
simplified regexp and fix grammar error

### PR DESCRIPTION
**Pull request by cl2017**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fix grammar error

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
Please have a look at regex changes, the editor show "\" is redundant, I tried to delete it, and didn't make a difference. Whether we should delete it?

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/util/common.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ import (
 )
 
 var (
-	kubeReleaseRegex = regexp.MustCompile(`^v?(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)([-0-9a-zA-Z_\.+]*)?$`)
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/util/common.go:49]: Why delete the `\` here?  We just import it from kubernetes, and it's also changed in K8s, ref to this patch: https://github.com/kubernetes/kubernetes/pull/112241    @cl2017

Comment (cl2017) [reply keadm/cmd/keadm/app/cmd/util/common.go:49]: ok, I had changed it from k8s

**Comment by cl2017** [comment]
@fisherxu I've already changed it, please have a look

**Comment by Iceber** [comment]
lgtm, But it needs to be organized for commits

**Pull request merged**